cypress/integration/search_for_property.spec.js

```diff
@@ -71,12 +67,37 @@ describe('Search for property', () => {
     cy.route('GET', 'api/v2/properties/?q=pitcairn', '@propertiesList')
 
     // Search by address
-    cy.get('.lbh-header__title-link').click()
     cy.get('.govuk-input').type('pitcairn')
     cy.get('[type="submit"]').contains('Search').click()
 
     cy.get('.govuk-heading-s').contains(
       'We found 2 matching results for: pitcairn'
     )
   })
+
+  it('Search for property with unraisable repair', () => {
```

Comment:
Maybe '... with unraisable tenure code'?

Reply:
I had a think about this and was wondering what  does the unraisable refer to if it's not with a repair context. But I suppose the app name makes it implicit 😁

Reply:
I wonder if this is telling us we'd benefit from splitting out a new integration spec. 

The last couple of specs here don't perform searches. What do you think about having a `show_property.spec.js` or similar?  Then we could have a `show property with a tenure type that permits repair raising` and its opposite. We might also want to bring some of the more detailed checks from 'Search for property' into its own spec there too. I'm thinking it would be a bit more readable / concise to have our search spec just input search terms, do some checks on the table results and then a check that the link in the table takes us to the show page, but no more. The rest is left to our show page spec. What do you think?

Reply:
Hey thanks for the suggestion @NeilvB . It makes a lot of sense, I'm definitely in agreement with you. Will make those changes 👍